### PR TITLE
Error message doesn't show up unless the whole page is loaded.

### LINKED ISSUE #2297: Error message when clicking verification link
**Describe the bug**
When a user clicks the verification link, error message highlights for a couple of seconds when the page loads and success message overrides it after the loading completes, which can be confusing. 

**Expected behavior**
The message displayed should be correct the first time.

**Additional context**
Working on it

### PR DESCRIPTION
**Pull request by prateekj117**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
On clicking the verification link sent to the email, the page which loads shows Error while loading, and success message after the page loads.

Fixes #2297

### CODE REVIEW

**Comment by prateekj117** [comment]
@CosmicCoder96 Please review this.

**Comment by CosmicCoder96** [comment]
Always add a small description in the PR of what it solves, please follow the PR template.

**Comment by prateekj117** [comment]
@CosmicCoder96 Please review now.

File: app/templates/verify.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="ui icon info message eight wide column center aligned">
     <p>Your email has been verified successfully! Please login to continue.</p>
   </div>
-{{else}}
+{{else if (not success)}}
```
Comment (CosmicCoder96) [inline app/templates/verify.hbs]: if success if false, wouldn't `not success` be true by default? What is the significance of this?

Comment (prateekj117) [reply app/templates/verify.hbs]: Else without a specified condition was printing until it was checking the status of "success", now it won't.

Comment (CosmicCoder96) [reply app/templates/verify.hbs]: okay, this seems like a bad implementation when it was first created, I will get back to you about this with a better fix, this shouldn't be happening in the first place.

Comment (prateekj117) [reply app/templates/verify.hbs]: @CosmicCoder96 Any update on this?

Comment (CosmicCoder96) [reply app/templates/verify.hbs]: @prateekj117 Yes, go to the controller of the `verify` route. Introduce the missing `isLoading` state variable in it. Set it to true when the `verify` function is called. In the template display a loading state until `isLoading` is set to false in the `finally` clause of the request in `verify` function.

**Comment by prateekj117** [comment]
@CosmicCoder96 Please review.

**Pull request merged**